src/types/UseReactToPrintOptions.ts

```diff
@@ -1,5 +1,5 @@
 import { RefObject } from "react";
-import type { Font } from "./Font";
+import type { Font } from "./font";
```

Comment:
Is this change intentional?

Reply:
There is only a "font.ts", no "Font.ts". My build failed with 'from "./Font"...

Reply:
I think my computer is lying to me!

![Screenshot 2024-08-19 at 1 51 58 AM](https://github.com/user-attachments/assets/6297f64c-6a2d-4cd2-9bc0-2fda36e231d4)
![Screenshot 2024-08-19 at 1 52 09 AM](https://github.com/user-attachments/assets/703874bc-51f4-483b-a2b4-9ab1abfa4940)